examples/plotting/file/customjs_hover.py

```diff
@@ -1,3 +1,12 @@
+''' An map of north Africa and south Europe with three interactive location points. When hovering over the points,
+its lat-lon is shown. This example demonstrates using CustomJSHover model and HoverTool to customize the formatting of values in tooltip fields.
```

Comment:
Having long metadata lines is often unavoidable, but we do try to keep the plain text parts of docstrings to 80 characters, so it would be preferred to shorten these lines a bit.